### PR TITLE
Web n android

### PR DESCRIPTION
**Pull request by MSiddharthReddy**
**What**:
* More android oprimizations.
* WEb raster is up and running with syncState.

**Why**:
* Seed up re-renders on android.
* Get web to parity

**How**:

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [x] Ready to be merged 
- [x] Added myself to contributors table

### CODE REVIEW

File: android/syrnative/src/main/java/syr/js/org/syrnative/SyrBridge.java [hunk 1]
```diff
@@ -113,7 +117,7 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
                 ///random comment
 
                 String screenDensity = Float.toString(mContext.getResources().getDisplayMetrics().density);
-                String loadURL = String.format("http://192.168.2.4:8080?window_height=%s&window_width=%s&screen_density=%s&platform=android&platform_version=%s&exported_methods=%s&initial_props",
+                String loadURL = String.format("http://10.0.0.2:8080?window_height=%s&window_width=%s&screen_density=%s&platform=android&platform_version=%s&exported_methods=%s&initial_props=%s",
```
Comment (shermango) [inline android/syrnative/src/main/java/syr/js/org/syrnative/SyrBridge.java]: what is this ip used for?

File: android/syrnative/src/main/java/syr/js/org/syrnative/SyrBridge.java [hunk 2]
```diff
@@ -171,6 +176,13 @@ public void runCMD(String commandString) throws JSONException, ClassNotFoundExce
                 }
             }
 
+            Log.i("Class name", c.toString());
```
Comment (shermango) [inline android/syrnative/src/main/java/syr/js/org/syrnative/SyrBridge.java]: comment out Log's

File: index.js [hunk 3]
```diff
@@ -40,6 +40,7 @@ import { NoDOM } from './lib/rasters/nodom';
 
 // central SyrStore (flux Store)
 import { SyrStore } from './lib/store';
+import { Resolver } from './lib/resolver';
```
Comment (shermango) [inline index.js]: where is this component coming from?

Comment (MSiddharthReddy) [reply index.js]: good catch, this is for the next SyrStore that I did not push yet!

File: android/syrnative/src/main/java/syr/js/org/syrnative/SyrBridge.java (hunk 2 above)
Comment (MSiddharthReddy) [reply android/syrnative/src/main/java/syr/js/org/syrnative/SyrBridge.java]: sure!

File: android/syrnative/src/main/java/syr/js/org/syrnative/SyrBridge.java (hunk 1 above)
Comment (MSiddharthReddy) [reply android/syrnative/src/main/java/syr/js/org/syrnative/SyrBridge.java]: this is where the WkWebkit looks for to get the AST. good catch though, it should be 10.0.2.2 for android localhost

**Comment by MSiddharthReddy** [comment]
@shermango Ran formatting on the android Code..!!! Could you review
 it again. Thanks!

File: android/syrnative/src/main/java/syr/js/org/syrnative/SyrInstance.java [hunk 4]
```diff
@@ -1,5 +1,7 @@
 package syr.js.org.syrnative;
 
+import android.content.Context;
```
Comment (shermango) [inline android/syrnative/src/main/java/syr/js/org/syrnative/SyrInstance.java]: unused import

File: android/syrnative/src/main/java/syr/js/org/syrnative/SyrTouchableOpacity.java [hunk 5]
```diff
@@ -18,55 +23,60 @@
...
 
             SyrStyler.styleView(layout, style);
 
 
             layout.setOnClickListener(new View.OnClickListener() {
-                @Override public void onClick(View v) {
+                @Override
+                public void onClick(View v) {
                     try {
+                        //@TODO: Make predefined vibrations like iOS and connect this to a prop
```
Comment (shermango) [inline android/syrnative/src/main/java/syr/js/org/syrnative/SyrTouchableOpacity.java]: we can define these as enum values

File: lib/button.js [hunk 6]
```diff
@@ -5,20 +5,22 @@ class Button extends Component {
   tag(instance) {
     let button = instance || document.createElement('button');
 
+    console.log(button);
```
Comment (shermango) [inline lib/button.js]: remove console.log here

File: android/syrnative/src/main/java/syr/js/org/syrnative/SyrTouchableOpacity.java (hunk 5 above)
Comment (MSiddharthReddy) [reply android/syrnative/src/main/java/syr/js/org/syrnative/SyrTouchableOpacity.java]: yeah, thats the idea!

**Pull request merged**